src/styles/main.css

```diff
@@ -0,0 +1,100 @@
+@import url('https://fonts.googleapis.com/css2?family=Gothic+A1:wght@100;200;300;400;500;600;700;800;900&family=Lato:wght@100;300;400;700&family=Montserrat:wght@100;200;300;400;500;600&family=Poppins:wght@100;200;300;400;500;600&family=Space+Grotesk:wght@300;400;500;600;700&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: 'Poppins', sans-serif;
+  box-sizing: border-box;
+}
+
+.container {
+  width: 100%;

```

Comment:
In following the best CSS practices, please make sure that there is no redundancy or duplication of CSS key-value pairs in our work. Let's make sure that we are organized in a way that ensures our code readability and maintainability. Example: if our CSS has **.container  set to wdith: 100% and #wrapper is set to width: 100%**, make sure that we have a single selector assigned to all this elements and give them a width of 100%. This ensures code readability and maintainability.